carbon-design-system/carbon-addons-iot-react
# fix(suite-header): fix keyboard keyboard navigation in header panel

Closes #2324 

**Summary**

- fixes keyboard navigation in the application switcher by adding keyDown handlers
- fixes nested A tags by changing the inner A tag to a Button.

**Change List (commits, features, bugs, etc)**

- add onKeyDown handlers for keyboard navigation
- add keyboard navigation tests

**Acceptance Test (how to verify the PR)**

- open the default SuiteHeader story
- click on the 9-dot menu to open to the application switcher panel
- tab navigate to the monitor or health links
- press enter or space to trigger those links
- confirm links take you to the new page

**Regression Test (how to make sure this PR doesn't break old functionality)**

- no other changes to suiteheader.
